cast/src/lib.rs

```diff
@@ -349,6 +349,12 @@ where
             "0x6341fd3daf94b748c72ced5a5b26028f2474f5f00d824504e4fa37a75767e177" => "rinkeby",
             "0xbf7e331f7f7c1dd2e05159666b3bf8bc7a8a3a9eb1d518969eab529dd9b88c1a" => "goerli",
             "0x14c2283285a88fe5fce9bf5c573ab03d6616695d717b12a127188bcacfc743c4" => "kotti",
+            "0xa9c28ce2141b56c474f1dc504bee9b01eb1bd7d1a507580d5519d4437a97de1b" => {
```

Comment:
just noticed that the naming for mainnet differs from ethers-rs

https://github.com/gakonst/ethers-rs/blob/19671e51deefec4c8f384270dd9141290a4f65b9/ethers-core/src/types/chain.rs#L95-L96

we should probably support both variants there

Reply:
`polygon` seems to be an outlier without `-mainnet`